### ISSUE TITLE
@types packages causing false positives 

### ISSUE THREAD

**Issue by sgen** [depcheck/depcheck#210]
[As of TypeScript 2.0 type definitions of pre-existing packages are available via `@types/pkg`](http://stackoverflow.com/questions/37548066/typescript-typings-in-npm-types-org-packages#answer-39998720).

`depcheck` should be able to recognize this.

Simply noting the packages that being with `@types/` and checking them against a list of used packages with the corresponding package name should do it.

Output of `depcheck --json`:
```
{
  "dependencies":[

  ],
  "devDependencies":[
    "@types/jasmine",
    "@types/jquery",
    "jasmine",
    "karma-chrome-launcher",
    "karma-jasmine",
    "karma-typescript"
  ],
  "missing":{

  },
  "using":{
    "gulp":[
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/gulpfile.js",
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/package.json"
    ],
    "gulp-tslint":[
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/gulpfile.js"
    ],
    "gulp-typescript":[
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/gulpfile.js"
    ],
    "karma":[
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/gulpfile.js"
    ],
    "del":[
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/gulpfile.js"
    ],
    "tslint":[
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/gulpfile.js"
    ],
    "typescript":[
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/gulpfile.js"
    ],
    "jquery":[
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/lib/nav.js",
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/src/nav.test.ts",
      "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/src/nav.ts"
    ]
  },
  "invalidFiles":{
    "/home/sgen/Documents/typescript/github.com/KunoSolutions/kn-nav/lib/nav.d.ts":"Error: Debug Failure. False expression: Output generation failed\n    at Object.assert (/usr/lib/node_modules/typescript/lib/typescript.js:3259:23)\n    at transpileModule (/usr/lib/node_modules/typescript/lib/typescript.js:76181:18)\n    at Object.transpile (/usr/lib/node_modules/typescript/lib/typescript.js:76189:22)\n    at parseTypescript (/usr/lib/node_modules/depcheck/dist/parser/typescript.js:24:27)\n    at /usr/lib/node_modules/depcheck/dist/check.js:129:17\n    at tryToString (fs.js:449:3)\n    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:436:12)"
  },
  "invalidDirs":{

  }
}
```

**Comment by rumpl** [comment]
Hey @sgen thanks for the issue, would you want to make a PR for this?

**Comment by sgen** [comment]
I might, very busy atm so I can make no promises.

**Comment by rjmunro** [comment]
This seems to be a duplicate of #163.

**Comment by mnkhouri** [comment]
Closing as duplicate of #163 -- we'll get this in for the 1.0 release
